components/prism-naniscript.js

```diff
@@ -0,0 +1,189 @@
+var expressionDef = function (isMultiline) {
+	if (isMultiline === void 0) { isMultiline = true; }
+	return isMultiline
+		? /\{[^\r\n\[\]{]*\}(?=[^\r\n\[\]\}]*)/m
+		: /\{[^\r\n\[\]{}]*\}/;
+};
+
+Prism.languages.naniscript = {
+	// ; ...
+	'comment': {
+		pattern: /^([\t ]*);[^\r\n]*$/m,
+		lookbehind: true,
+	},
+	// > ...
+	// Define is a control line starting with '>' followed by a word, a space and a text.
+	'define': {
+		pattern: /^>.+$/m,
+		alias: 'tag',
+		inside: {
+			value: {
+				pattern: /(^>[a-zA-Z0-9_]+)[\t ]+[^{}\r\n]+/,
```

Comment:
We generally don't want tokens to start or end with spaces. There are cases where that's hard to do but here it's easy.

```suggestion
				pattern: /(^>[a-zA-Z0-9_]+[\t ]+)[^{}\r\n]+/,
```